build-system/tasks/serve.js

```diff
@@ -129,7 +129,7 @@ async function startServer(
   log(green('Started'), cyan(options.name), green('at:'));
   log('\t', cyan(url));
   for (const device of Object.entries(os.networkInterfaces())) {
-    for (const detail of device[1]) {
+    for (const detail of device[1] ?? []) {
```

Comment:
The readability here isn't 100% - how about `for (const detail of (device[1] ?? []))`?